### PR TITLE
Networkpolicy handles pod update/delete

### LINKED ISSUE #339: Networkpolicy handles pod update

### LINKED ISSUE #350: Networkpolicy handles pod delete

### PR DESCRIPTION
**Pull request by Hong-Chang**
In prev merge, when a pod update/delete, a label list is generated. This change is to check whether the changed labels are connected to network policies. If yes, trigger all the endpoints data process for those affected network policies.

Also removed networkpolicy object. For now it's not needed.

### CODE REVIEW

File: mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 @kopf.on.create('networking.k8s.io', 'v1', 'networkpolicies', retries=OBJ_DEFAULTS.kopf_max_retries, when=LAMBDAS.k8s_provider_vanilla)
 async def builtins_on_networkpolicy(body, spec, **kwargs):
     param = HandlerParam()
-    param.name = "{}:{}".format(kwargs['namespace'],kwargs['name'])
```
Comment (clu2xlu) [inline mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py:37]: Why are you changing param.name?

Comment (clu2xlu) [reply mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py:37]: namespace should be part of body: param.body['metadata']

https://github.com/CentaurusInfra/mizar/blob/7112bc78c8e9d2a175bbe3067793d3258da179cf/mizar/dp/mizar/workflows/builtins/pods/create.py#L58

**Review by clu2xlu — CHANGES_REQUESTED** [review]

File: mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py (hunk 1 above)
Comment (Hong-Chang) [reply mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py:37]: I just add a space here when I notice the tiny format issue. The code was working.

Comment (clu2xlu) [reply mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py:37]: I am actually asking you to change to param.name = kwargs['name']
And if you want get the namespace of the network policy, you can use param.body['metadata']['namespace']

**Review by clu2xlu — APPROVED** [review]

**Pull request merged**